src/Collections-Abstract-Tests/CollectionRootTest.class.st

```diff
@@ -125,6 +125,29 @@ CollectionRootTest >> testDetectIfFoundWhenSomethingIsFoundIgnoringTheFoundObjec
 	self assert: wasFound
 ]
 
+{ #category : #'tests - iterating' }
+CollectionRootTest >> testDo2 [
```

Comment:
This test popped up while doing the automatic rename from `FloatArray` to `Float32Array`. Should we remove it?

Reply:
Yes. Tests are indeed failing because of that. I'll delete them